regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions_by_literal.groovy

```diff
@@ -190,6 +190,6 @@ suite("test_array_functions_by_literal") {
     // abnormal test
     test {
         sql "select array_intersect([1, 2, 3, 1, 2, 3], '1[3, 2, 5]')"
-        exception "errCode = 2, detailMessage = No matching function with signature: array_intersect(array<tinyint(4)>, varchar(-1))"
+        exception "errCode = 2, detailMessage = No matching function with signature: array_intersect(array<tinyint(4)>, varchar(65533))"
```

Comment:
For this case, we maybe only need to catch error msg prefix: `errCode = 2, detailMessage = No matching function with signature: array_intersect`

Reply:
you're right, but the pipeline is blocked， It's just a quick fix ，I will fix it in latter PR

Reply:
fixed